client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -922,7 +922,7 @@ export class PlanFeatures2023Grid extends Component<
 
 				const shouldRenderStorageTitle =
 					storageOptions.length === 1 ||
-					( intervalType === 'monthly' && storageOptions.length !== 0 ) ||
+					( intervalType !== 'yearly' && storageOptions.length !== 0 ) ||
```

Comment:
Nit/Unrelated any idea why we don't use  `storageOptions.length > 0` ?

Reply:
In its current implementation, storageOptions data structure now contains the storage feature inherent to the plan ( Ex. 1GB, 6GB, etc. ) _and_ storage add ons ( Ex. 50GB add-on, 100GB add-on )

Reply:
Good question! That would work as well and is more explicit.

Let me double check that there isn't something simple I'm overlooking.

Reply:
[456bb1f](https://github.com/Automattic/wp-calypso/pull/80155/commits/456bb1fcc3a5655a87b376ed5b90598ff3d4c148)